applications/serial_lte_modem/doc/TCPUDP_AT_commands.rst

```diff
@@ -207,6 +207,10 @@ Syntax
 
 See :ref:`nRF socket options <nrfxlib:nrf_sockets>` ``peer_verify`` and ``tls_hostname`` for more information on ``<peer_verify>`` and ``<hostname_verify>``.
 
+* The ``<cid>`` parameter is an integer indicating the used PDN connection.
+  It represents ``cid`` in the ``+CGDCONT`` command.
+  Its default value is ``0``.
```

Comment:
While default value can be considered zero, we don't bind the TCP/UDP client to default PDN when zero or nothing is given.
So when 'cid' is not given the question is, does the stack handle everything like it was explicitly bound to PDN 0?

Reply:
SO_BINDTOPDN does not support GET but on modem side apparently yes, check +CGACT note

Reply:
I don't know if this is 100% accurate but conceptually I guess this is fine.